app-service-template/res/configuration.toml

```diff
@@ -0,0 +1,102 @@
+
+    # TODO: if not running in secure mode, but do have secrets then add them here.
+    [Writable.InsecureSecrets]
+        [Writable.InsecureSecrets.DB]
+        path = "redisdb"
+            [Writable.InsecureSecrets.DB.Secrets]
+            username = ""
+            password = ""
+
+[Service]
+BootTimeout = '30s'
+ClientMonitor = '15s'
+CheckInterval = '10s'
+Host = 'localhost'
+Port = 49600  # TODO: set this port appropriatly
+Protocol = 'http'
+ReadMaxLimit = 100
+StartupMsg = 'new-app-service Application Service has started'
+Timeout = '30s'
+
+[Registry]
+Host = 'localhost'
+Port = 8500
+Type = 'consul'
+
+[Database]
+Type = "redisdb"
+Host = "localhost"
+Port = 6379
+Timeout = "30s"
+
+# TODO: Determine if your service will use secrets in secure mode, i.e. Vault.
+#       if not this secion can be removed, but you must make sure EDGEX_SECURITY_SECRET_STORE is set to false
+#       Note is database is running in secure more and you have Store and Forward enable you will need to run this
+#       service in secure mode.
+#       For more deatils about SecretStore: https://docs.edgexfoundry.org/1.3/microservices/security/Ch-SecretStore/
+[SecretStore]
+Host = 'localhost'
+Port = 8200
+Path = '/v1/secret/edgex/appservice/'
+Protocol = 'http'
+RootCaCertPath = ''
+ServerName = ''
+TokenFile = '/vault/config/assets/resp-init.json'
+AdditionalRetryAttempts = 10
+RetryWaitPeriod = "1s"
+    [SecretStore.Authentication]
+    AuthType = 'X-Vault-Token'
+
+[Clients]
+  [Clients.CoreData]
+  Protocol = 'http'
+  Host = 'localhost'
+  Port = 48080
+
+[Binding]
+Type="edgex-messagebus"
+SubscribeTopic="events"
+PublishTopic="event-xml"  #TODO: remove if service is NOT publishing back to the message bus
+
+[MessageBus]
+Type = 'zero'
+    [MessageBus.SubscribeHost]
+        Host = 'localhost'
+        Port = 5563
+        Protocol = 'tcp'
+    [MessageBus.PublishHost]   # TODO: Remove if service is NOT publishing back to the message bus
+        Host = '*'
+        Port = 5564
+        Protocol = 'tcp'
+
+# TODO: If using mqtt messagebus, Uncomment this section and remove above [Binding] & [MessageBus],
+#       Otherwise remove this commentedout block
+#[Binding]
+#Type="edgex-messagebus"
+#SubscribeTopics="events"
+#PublishTopic="event-xml"  # TODO: Remove if service is NOT publishing back to the message bus
+#
+#[MessageBus]
+#    Type = 'mqtt'
+#    [MessageBus.SubscribeHost]
+#    Host = 'localhost'
+#    Port = 1883
+#    Protocol = 'tcp'
+#    [MessageBus.PublishHost]   # TODO: Remove if service is NOT publishing back to the message bus
+#        Host = 'localhost'
+#        Port = 1883
+#        Protocol = 'tcp'
+
+[ApplicationSettings]
+# TODO: Add custom settings needed by your app service
+# This can be any Key/Value pair you need.
+# Example that works with devices from the Virtual Device service:
+DeviceNames = "Random-Boolean-Device, Random-Integer-Device, Random-UnsignedInteger-Device, Random-Float-Device, Random-Binary-Device"
```

Comment:
This isn't super clear... Are settings under this stanza passed through to the application and thus ignored by the app functions code (i.e. this is basically equivalent to [Driver] section in a device service? If so, a pointer to the relevant documentation might be helpful.

Reply:
Yes, this is the same as `[Driver]` in Device Services. 
Great suggestion on the docs link. Added.